.github/CODEOWNERS

```diff
@@ -3,5 +3,3 @@
 
 
 @hashicorp/consul-ecs
-
-/.github/workflows/* @hashicorp/release-engineering @hashicorp/consul-ecs
```

Comment:
Team hashicorp/release-engineering is now `hashicorp/team-selfmanaged-releng` and`team-cloud-releng`. We just need to pick if this repo should be managed by cloud or selfmanaged

Reply:
Also not sure if `consul-ecs` is a valid team

Reply:
I think we want this file to be:
```
* @hashicorp/consul-selfmanage-maintainers

# release configuration
/.github/workflows/*            @hashicorp/team-selfmanaged-releng @hashicorp/consul-selfmanage-maintainers

```

@NiniOak  maybe you can make a new PR and we can ask that this one not get merged?